io.github.lucasew.regex101.yaml

```diff
@@ -0,0 +1,33 @@
+id: io.github.lucasew.regex101
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-23.08
+command: io.github.lucasew.regex101
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents/regex101.sqlite
+modules:
+  - name: build
```

Comment:
please use a descriptive name. This is a module not a "step".

Reply:
changed